util/config.go

```diff
@@ -329,17 +334,44 @@ func (config *Config) getAndroidManifestPath() string {
 }
 
 func (config *Config) getAndroidConfigModelPath() string {
-	return path.Join(getPlatformSpecificAndroidClasspathPath(config), "OneginiConfigModel.java")
+	modelPath := path.Join(getPlatformSpecificAndroidClasspathPath(config), "OneginiConfigModel.java")
+	// if modelPath has no package name, check namespace property in build.gradle
+	if strings.HasSuffix(modelPath, "java/OneginiConfigModel.java") {
```

Comment:
Checking suffix of the path if it contains package name. Correct one would be  java/com/my/example/OneginiConfigModel.java